### DISCUSSION TITLE
Canvas self inserts, all-seeing-eye rotation

### DISCUSSION THREAD

**Discussion by VegaDeftwing** [noisedeck/noisedeck-docs#26] [Ideas]
It'd be interesting to be able to do feedback with the canvas itself duplicated and rendering.. inside of itself? I don't have a better way to phrase that. See this tweet I sent out to Alex: https://twitter.com/Vega_DW/status/1415036099685138439

From the same thought- it would be cool if the view from all seeing eye could be rotated along with the current options of changing X, Y, and Scale.

**Comment by jesshewitt** [comment]
Interesting idea! Shouldn't be hard to add rotation to all seeing eye. Will have to do some experimentation to see how canvas self-feedback might work.
